website/pages/docs/platform/k8s/injector/annotations.mdx

```diff
@@ -126,6 +135,17 @@ them, optional commands to run, etc.
   ~> **Note**: If the first application container in the pod is running as root
   (uid 0), the `run-as-same-user` annotation will fail injection with an error.
 
+- `vault.hashicorp.com/agent-cache-enable` - configures Vault Agent to enable 
+  [caching](/docs/caching).  Defaults to `false`.
+
+- `vault.hashicorp.com/agent-cache-use-auto-auth-token` - configures Vault Agent cache 
+  to authenticate on behalf of the requester.  Set to `force` to enable.  Disabled 
```

Comment:
You've got a double space at the start of the new sentence here and a couple of other places. Not sure if intentional?

Reply:
Was taught to use two spaces after a period, bad habit.  Fixed them!